api/swagger.yaml

```diff
@@ -8068,6 +8075,13 @@ paths:
             Docker-Experimental:
               type: "boolean"
               description: "If the server is running with experimental mode enabled"
+            Swarm:
+              type: "string"
+              enum: ["inactive", "pending", "error", "locked", "active (worker)", "active (manager)"]
```

Comment:
"worker" could be sufficient to mean "active (worker)", same for "manager". Looks easier to define and to parse in client side. What do you think?